rpcd/playbooks/ceph-mon.yml

```diff
@@ -17,8 +17,65 @@
   hosts: mons
   user: root
   max_fail_percentage: 0
+  #serial: "{{ ceph_upgrade | default(False) | bool | ternary('1','100%') }}"
+  pre_tasks:
+    #Upgrades safety net
+    - name: Checking if container has ceph content
+      stat:
+        path: "/var/lib/ceph/"
+      register: container_ceph_dir
+      failed_when: false
+
+    - name: Checking if the ceph folder is a bind mount
+      shell: egrep 'lxc.mount.entry.*/openstack/{{ inventory_hostname }}.*ceph.*bind' /var/lib/lxc/{{ inventory_hostname }}/config
+      failed_when: false
+      delegate_to:  "{{ physical_host }}"
+      when: container_ceph_dir is defined
+      register: container_ceph_bind_mount
+
+    - name: If ceph is not in a bind-mount, create a backup folder
+      file:
+        state: directory
+        path: /backup/
+      when: container_ceph_bind_mount | failed
```

Comment:
I don't think so, the container_ceph_bind_mount wouldn't be failed if the folder exists, it would simply be skipped.